test/jasmine/tests/sankey_test.js

```diff
@@ -260,6 +260,12 @@ describe('sankey tests', function() {
 
         afterEach(destroyGraphDiv);
 
+        function wait() {
+            return new Promise(function(resolve) {
+                setTimeout(resolve, 60);
+            });
+        }
```

Comment:
That thing got :hocho: 

I've been using this [pattern](https://github.com/plotly/plotly.js/pull/1582/files#diff-846cf5b534aa0d22bdd1da2b43ac3cbaR453) lately. No need for timeouts :tada: